src/app/content/highlights/components/SummaryPopup/FiltersList.tsx

```diff
@@ -125,8 +125,12 @@ export default styled(FiltersList)`
   flex-wrap: wrap;
   width: 100%;
   margin: 0;
-  padding: 0.4rem 0;
+  padding: 0 2.8rem 1rem 2.8rem;
+  margin-top: -1rem;
   list-style: none;
+  ${theme.breakpoints.mobile(css`
+    padding: 0 2.4rem 0.4rem 2.4rem;
```

Comment:
a little bit of inline numbers is understandable, but you've got a lot of them, i feel like a bunch should be derived from constants